lib/cadet_web/admin_controllers/admin_assessments_controller.ex

```diff
@@ -0,0 +1,147 @@
+defmodule CadetWeb.AdminAssessmentsController do
+  use CadetWeb, :controller
+
+  use PhoenixSwagger
+
+  alias Cadet.Assessments
+  import Cadet.Updater.XMLParser, only: [parse_xml: 2]
+
+  def create(conn, %{"assessment" => assessment, "forceUpdate" => force_update}) do
+    file =
+      assessment["file"].path
+      |> File.read!()
+
+    result =
+      case force_update do
+        "true" -> parse_xml(file, true)
+        "false" -> parse_xml(file, false)
+      end
```

Comment:
Just checking, forceUpdate is a string because this is submitted as multipart/form-data, right?

Reply:
Yeah, this is the case.